openapi/components/schemas/CurrentAvatarImageUrl.yaml

```diff
@@ -0,0 +1,4 @@
+description: 'When `profilePicOverride` is set, use it insted.'
```

Comment:
Rather than "is set", it should be "is not empty". As the value is always present, but is an empty string.

Reply:
*instead

Reply:
Thank you!

I didn't get good English.
Is this okay?

> 'When `profilePicOverride` is not empty, use it.'

Reply:
"use it instead", Yes. 👍

Dont worry, your English was good, it was just a typo "insted" should be "instead".

Reply:
I see. Thank you, and please check again.